utils/form-element.js

```diff
@@ -67,6 +67,26 @@ class FormElement {
 	value () {
 		return this.$el.querySelector('input,select').value;
 	}
+
+	/**
+	 * Make firts input element required
+	 */
+	required () {
```

Comment:
Wit the current method name, I would expect this to return if a field is **required** or **not**
```suggestion
	makeRequired () {
```

Reply:
+1 on this, I find it a bit ambiguous.
I think we should try to make things as explicit as possible, like `isRequired()`, `makeRequired()`, etc

Reply:
Agree, additionally I have added a new suite of tests that cover those methods using company-name component

Reply:
92577ce408b16a7559fa8afcb2aba3739fa80fc0